cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -224,9 +224,9 @@ protected void shellHandleIntent(Intent intent) {
       }
   }
 
-  // TODO(cobalt, b/376148547): set Chrobalt initial url and remove this function.
-  protected void setStartupUrl(String url) {
-      mStartupUrl = url;
+  protected void toggleFullscreenMode(boolean enterFullscreen) {
+    LinearLayout toolBar = (LinearLayout) findViewById(R.id.toolbar);
```

Comment:
Can findViewById() return null?

Reply:
It could be, let me gate it.

Reply:
It won't, as this is called after setContentView, and we using the same layout as ContentShellActivity, so the toolbar will always exists.